hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcColumnTypeNameResolver.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.connector.jdbc;
+
+import com.hazelcast.sql.impl.type.QueryDataType;
+
+final class JdbcColumnTypeNameResolver {
+    private JdbcColumnTypeNameResolver() {
+    }
+
+    /**
+     * Convert the column type received from database to QueryDataType. QueryDataType represents the data types that
+     * can be used in Hazelcast's distributed queries
+     */
+    @SuppressWarnings("ReturnCount")
+    public static QueryDataType resolveType(String columnTypeName) {
```

Comment:
Please move this back to the `JdbcSqlConnector`. 

In general please don't move out private (static) methods out to util classes unless you need to call them from multiple classes.

Reply:
rev : 5750495eedfd37a0c15e484adff25c12604d0002